Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -1604,6 +1741,29 @@ def convert_start_fetch_to_milliseconds(fetch_start_time: str):
         raise ValueError(f'Invalid first_fetch format: {fetch_start_time}')
     return int(date.timestamp() * 1000)
 
+def convert_dict_values_string_to_number(obj: dict) -> dict:
+    clone = { **obj }
+    for key, value in obj.items():
+        if isinstance(value, str) and value.isnumeric():
+            clone[key] = int(value)
+    return clone
+
+def convert_to_actual_values_recursive(input_dict: dict) -> dict[str, Any]:
```

Comment:
I would be a bit cautious with this function,

what if for example one of the parameters accepts a string and the user passes "1234". For example, description and the user passes in a numeric value. The api would expect a string, but would receive an int. This might break things. 

Lets double check 2 things - 1 that the api cant recieve strings in these instances, and 2 if it cant, lets check that there arent cases like the one i mentioned above, and if there are, can the api handle this?

Reply:
what do you think about the proposed solution?